### PR TITLE
Setup project files

### PR DESCRIPTION
**Pull request by ishimwepati**
In this branch I did :

- I added a HTML file
- I also added a CSS file
- I Edited the ReadMe File
- Also added Linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,183 @@
...
+> - Added CSS
+> - Added Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+LIVE DEMO
+
+> Link to Live Demo will be added soon.
```
Comment (stanleySimeon) [inline README.md]: I suggest you remove the live demo section if your project does not have a live demo link. Note that you can add it when the live demo link is available. 😄

File: README.md [hunk 2]
```diff
@@ -1,2 +1,183 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> To get a local copy up and running, follow these steps.
```
Comment (stanleySimeon) [inline README.md:75]: I suggest you add the steps to get a local copy of your repo to make your project look more professional. 😄

File: README.md [hunk 3]
```diff
@@ -1,2 +1,183 @@
...
+<!-- INSTALL -->
+
+### Install
+
+Install this project with the latest version of Git and a code editor.
+
+### Usage
+
+To run the project, execute the following command:
+> N/A
```
Comment (stanleySimeon) [inline README.md]: Also, add the command to run the project.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,183 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **Grades Management Portal**
+- [ ] **Fee Payment Portal**
+- [ ] **Student Assessment Portal**
+
```
Comment (stanleySimeon) [inline README.md]: I suggest you remove this section if you don't have to work on some features.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,183 @@
...
+  - We'll be working on adding **Student Assessment Portal**, **Grades Management Portal**, and **Fee Payment Portal**.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (stanleySimeon) [inline README.md:173]: I suggest you fix this link when the user clicks on it, should redirect to the `MIT` license. by doing that your project will look more professional. 😄

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @ishimwepati , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of Gitflow ✔️
- Linter checks are passed ✔️
- Good PR description ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- adding the `test.md` file in the PR for review.
- adding the `.gitignore` file in the PR for review.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (ishimwepati) [reply README.md:75]: Wonderful, let me do it !

File: README.md (hunk 1 above)
Comment (ishimwepati) [reply README.md]: Thank you for this suggestion.

File: README.md (hunk 3 above)
Comment (ishimwepati) [reply README.md]: Thanks

File: README.md (hunk 4 above)
Comment (ishimwepati) [reply README.md]: Yeah, i have removed them.

File: README.md (hunk 2 above)
Comment (ishimwepati) [reply README.md:75]: Wonderful, let me do it !

File: README.md (hunk 4 above)
Comment (ishimwepati) [reply README.md]: Yeah, i have removed them.

File: README.md (hunk 5 above)
Comment (ishimwepati) [reply README.md:173]: The linked is fixed

**Review by divinecharlotte — APPROVED** [review]
# Hello @ishimwepati    😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Descriptive pr👍 
- Linters are passing👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**